docs/filters.md

```diff
@@ -70,6 +73,144 @@ https://<webapp>/api/v1.1/mymodel/?creation_date__range=date1,date2
 ```
 
 
+<a name="TimestampStartEnd"></a>**Using timestamp_start and timestamp_end examples**:
+
+Given a model `Article` with a float field `price`.
+Given two instances of this model:
+
+-  article1 (price=53.99)
+-  article2 (price=52.99)
+
+To retrieve all articles with the price is greater that 50.00:
+```
+https://<webapp>/api/v1.1/article/?price__gte=50.0&timestamp_start=0
+```
+
+The API response is:
+
+```json
+{
+  "objects_count": 2,
+  "next": null,
+  "previous": null,
+  "results": [ // both articles are in the reponse
+    // article1,
+    // article2
+  ],
+  "objects_count_per_page": 250,
+  "num_total_pages": 1,
+  "num_current_page": 1,
+  "max_allowed_objects_per_page": 250,
+  "model_name": "Article",
+  "model_verbose_name": "Article",
+  "list_display": [],
+  "list_filter": {},
+  "total_objects_count": 2,
+  "create_url": "https://<webapp>/api/v1.1/article/",
+  "timestamp_start": 0.0,
+  "timestamp_end": 1603716926.382462,
+  "deleted_uids": []
+}
+```
+
+If you use the returned `timestamp_end` to perform the GET request:
+```
+https://<webapp>/api/v1.1/article/?price__gte=50.0&timestamp_start=1603716926.382462
+```
+
+then the results will be an empty list, because between `1603716926.382462` and the current timestamp nothing changed in the Database
```

Comment:
```suggestion
then the results will be an empty list, because between `1603716926.382462` and the current timestamp, nothing changed in the Database
```